site/en/docs/extensions/mv3/samples/index.md

```diff
@@ -1,9 +1,69 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Samples"
-#date: TODO
-#updated: TODO
-#description: TODO
+seoTitle: "Chrome Extension examples"
+date: 2020-12-11
+updated: 2022-12-16
+description: Intro to extension examples GitHub repository 
 ---
 
-Samples for Chrome Extensions are available [on GitHub](https://github.com/GoogleChrome/chrome-extensions-samples).
+## Overview {: #overview }
+
+The [Extensions sample GitHub
+repository][gh-samples] provides examples of extensions that address various use cases and call appropriate Chrome
+APIs. Use these to learn how extensions work or as a starting point for building your own extensions.
+
+## Browse by category {: #categories }
+
+The examples are currently located under the following directories:
+
+[api/][gh-api]
+: These extensions were designed to demonstrate the capabilities of a specific API. For example, the [Action API example][gh-action] showcases extension IU elements such as the popup, tooltip, and badges among others. 
+
+[examples/][gh-examples]
+: These are complete extensions that implement all the basic features for a given purpose.
+
+[tutorials/][gh-tutorials]
+: These examples include step-by-step instructions on how to build an extension. A few examples include [Tabs manager][tut-tabs-man], [Focus mode][tut-fm], and [Reading time][tut-rt].
+
+{% Details %}
+{% DetailsSummary %}
+💡 **TIP**: How can I find examples of a particular API quickly?
+{% endDetailsSummary %}
+
+To find all examples of a specific Chrome API use Github's search engine to look for "chrome.NAME_OF_API" in this repository. The following example searches for all uses of "chrome.tabs":
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/2OW6hjG5RLoupNCzZk8u.png", alt="Searching for chrome.tabs in the extension Github repository", width="800", height="122", class="screenshot screenshot--filled"  %}  <figcaption>
+Chrome.tabs search in the extension Github repository
+  </figcaption>
+</figure>
+
+{% endDetails %}
+
+## Try them out {: #testing }
+
+To test these examples in your local machine, follow these steps:
+
+1. Clone the repository.
+1. Navigate to the directory of the extension you want to try.
+1. [Load your extension locally][dev-basics-locally].
+
+Each extension example includes a `README.md` with detailed instructions on how the extension works.
+Read it carefully; each extension is different. For example, some extensions run by clicking on the extension icon, while others run automatically but only run on specific sites. 
+
+## Contribute to this repository {: #collaborating }
```

Comment:
"This repository" is a little unclear in this context. I think it would be best to have a more explicit section title.

```suggestion
## Contributing samples {: #collaborating }
```

Reply:
Thanks for this! Looks much better 👍 🙂